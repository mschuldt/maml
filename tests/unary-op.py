arduino = Arduino(desktop=True)

@block
def test():
    x = -1
    print_i(x) # -1
    x *= 5
    print_i(x) # -5
    x *= -1
    print_i(x) # 5
    x *= -6
    print_i(x) # -30
    die()

arduino.send(test)
