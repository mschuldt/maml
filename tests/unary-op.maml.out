Initializing avm...
Ready.

-1
-5
5
-30
bye.
